redis/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "redis"
-version = "0.23.0"
+version = "0.24.0-alpha"
```

Comment:
they'll probably ask you to keep the version as it is for now

Reply:
reverted